modules/cvv/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+set(the_description "Debug visualization framework")
+ocv_define_module(cvv opencv_core opencv_imgproc opencv_features2d ${CVV_LIBRARIES} WRAP python)
```

Comment:
This module requires QT5, so there 2 lines should be placed after next `if` block